snapcraft/plugins/python3.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015 Canonical Ltd
+# Copyright (C) 2015-2016 Canonical Ltd
```

Comment:
El 01/09/16 a las 13:01, Leo Arias escribió:
>
> In snapcraft/plugins/python3.py 
> <https://github.com/snapcore/snapcraft/pull/771#discussion_r77203058>:
>
> > @@ -1,6 +1,6 @@
> >  # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
> >  #
> > -# Copyright (C) 2015 Canonical Ltd
> > +# Copyright (C) 2015-2016 Canonical Ltd
>
> 2015, 2016
>
>
we already had this discussion ;-)